### DISCUSSION TITLE
cursorSmoothCaretAnimation

### DISCUSSION THREAD

**Discussion by Dingzhaocheng** [zed-industries/zed#6487] [Feature Requests]
#### Cursor smooth transition animation can make the visual effect smoother, faster and more comfortable, do we have plans to implement this feature

**Comment by JosephTLyons** [comment]
We've discussed adding animations to Zed.  I think we will likely do this, but not sure of when.

**Reply by mocenigo** [reply]
Triage well. For instance, the use of more than one language server simultanously on the same file and more extension API are more important. 

I have switched recently for all my editing to Zed, and despite the fact that I miss a couple of things, I love the responsiveness. You guys have chosen the right data structures for the text and implemented them well, and it is the fastest editor I know. And it would be even with a non-accelerated UI. The latter makes it "lickable" as Steve Jobs would put it. Not indispensable, perhaps, but it is part of the Zed experience.

**Comment by tahirmurata** [comment]
Any updates?

**Comment by JuZaixx** [comment]
Anything changes with this?

**Comment by gwynnnplaine** [comment]
Bump.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
